<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyResultImportanceForm</class>
 <widget class="GUIFormBase" name="PropertyResultImportanceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Property ResultImportance</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetAreaList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>381</width>
     <height>391</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Location</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Importance</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetImportanceList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>191</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Importance</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Count</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectLines">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Items with selected line</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyResultImportance.qrc">
     <normaloff>:/Qtres/Select.png</normaloff>:/Qtres/Select.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>265</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Importance Color</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbImportanceColor">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyResultImportance.qrc"/>
 </resources>
 <connections/>
</ui>
